core/shared/platform/include/platform_common.h

```diff
@@ -118,6 +125,26 @@ typedef void *(*thread_start_routine_t)(void *);
 #define bh_socket_t int
 #endif
 
+/**
+ * Get a resolution of the clock
+ *
+ * @param clock_id clock identifier
+ * @param resolution output variable to store the clock resolution
+ * @return BHT_OK if success; otherwise, BHT_ERROR
+ */
+int
+os_clock_res_get(bh_clock_id_t clock_id, uint64 *resolution);
```

Comment:
I'm actually wondering if we should move those two declarations to `platform_api_extension.h` file.